tests/scripts/thread-cert/node.py

```diff
@@ -1004,6 +1010,11 @@ def set_active_dataset(
             self.send_command(cmd)
             self._expect('Done')
 
+        # Set the meshlocal prefix in config.py
+        self.send_command('dataset meshlocalprefix %s' %
```

Comment:
Add as an argument `mesh_local_prefix=config.MESH_LOCAL_PREFIX` ?

Reply:
I tend not to extend the API until we really need it. Thoughts?